docs/rules/no-trailing-zeros.md

```diff
@@ -0,0 +1,48 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "yml/no-trailing-zeros"
+description: "disallow trailing zeros for numbers"
+since: "v1.5.1"
+---
+
+# yml/no-trailing-zeros
+
+> disallow trailing zeros for numbers
+
+- :gear: This rule is included in `"plugin:yml/standard"`.
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces the removal of unnecessary trailing zeros from numbers.
+
+<eslint-code-block fix>
+
+<!-- eslint-skip -->
+
+```yaml
+# eslint yml/no-trailing-zeros: 'error'
+
+# ✓ GOOD
+"GOOD": 1.2
+
+# ✗ BAD
+'BAD': 1.20
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+Nothing.
+
+## :rocket: Version
+
+This rule was introduced in eslint-plugin-yml v1.5.1
```

Comment:
Should it be deleted as well?

Reply:
It will be rewritten automatically when you run `npm run update`.